.github/workflows/dependency-review-action.yml

```
@@ -0,0 +1,153 @@
+# Automatically scans every PR for newly added dependencies
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)